### DISCUSSION TITLE
Feature: "Open in Tabs but it opens /blog/view"

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#564] [Suggestion Box]
Not sure what to call this exactly, but Open in Tabs carefully emulates the legacy behavior of "always open `[blogName].tumblr.com` instead of `tumblr.com/blog/view/[blogname]` on left click." I would like the opposite of this behavior, opening `tumblr.com/blog/view/[blogname]` on a left click of links which would normally go to the other one (specifically readmores in reblogs; I don't have a mental list of other places where this happens and whether I would actually want this behavior in those hypothetical places.)

If it only turns out to be readmores in reblogs, this doesn't make a whole lot of sense, so I will try to keep it in the back of my mind and comment if I find more places where this applies. I feel like there are more... I think. Maybe.

**Comment by AprilSylph** [comment]
This more broadly applies to any plain links within post content pointing to blogs/posts, but I believe reblogged legacy readmores are the only method by which they are created automatically.
